### DISCUSSION TITLE
VS Build Tools 17.10 Update

### DISCUSSION THREAD

**Discussion by lbussell** [microsoft/dotnet-framework-docker#1129] [Announcements]
# VS Build Tools 17.10 Update

Today's release of the .NET Framework images includes an update to the SDK images that upgrades the Visual Studio Build Tools version to 17.10.

## Affected Tags

All tags in the [`dotnet/framework/sdk` repository](https://mcr.microsoft.com/en-us/product/dotnet/framework/sdk/about) are updated with this change.

**Comment by bthharper** [comment]
There was a patch made to the release 17.10.1 (might be 4 now) - has this been upgraded?

**Reply by mthalman** [reply]
No. This will be updated as part of our normal monthly servicing schedule (2nd Tuesday of each month). In this case, that will be on June 11.

**Reply by bthharper** [reply]
Okay - we've been caught out by a build tools bug preventing us from building, is there a version that includes 17.9 of the build tools I could point at?

**Reply by mthalman** [reply]
Yes, there are timestamp tags available. You can see the diff between the old tags and the new tags that brought in this change here: https://github.com/microsoft/dotnet-framework-docker/pull/1127/files. You'd want the `20240514` timestamp. Since there hasn't been a Windows security update since before this change was made, those previous tags should be safe to use.
